### PR TITLE
Chore: small update to the README file

### LINKED ISSUE #13: Update documentation
Update Readme file

### PR DESCRIPTION
**Pull request by jesus4497**
Closes #13 

### Changes

- Added final API URL and missed image

### CODE REVIEW

**Review by mimurawil — APPROVED** [review]
This looks good to me 👍 

Maybe it's good to have some 👀  from @dustinsgoodman.

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Here's a list of our available API endpoints:
 
 ### GraphQL Queries
 
-GraphQL queries can be accessed using the endpoint `/graphql`. Open the sandbox at `<add final URL here>` and take a look at the query schema documentation.
+GraphQL queries can be accessed using the endpoint `/graphql`. Open the sandbox at `https://0kadddxyh3.execute-api.us-east-1.amazonaws.com/graphql` and take a look at the query schema documentation.
```
Comment (dustinsgoodman) [inline README.md]: Maybe note how the schema won't load without the access token auth header set?

Comment (dustinsgoodman) [reply README.md]: Never mind - I see it above.

File: README.md [hunk 2]
```diff
@@ -16,14 +16,14 @@ Welcome to the `movies-api` repository! This open-source project provides a comp
 
 To use our API, first, obtain a bearer token:
 
-1. Send a `GET` request to `<final URL here>/auth/token`.
+1. Send a `GET` request to `https://0kadddxyh3.execute-api.us-east-1.amazonaws.com/auth/token`.
```
Comment (dustinsgoodman) [inline README.md]: Maybe add the base URL at the top of this readme for folks. Otherwise this seems good to me.

**Review by dustinsgoodman — APPROVED** [review]

**Pull request merged**